test/client/end-to-end.clienttest.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
       .assert.urlEquals('http://localhost:8000/search?q=ada')
       .click('.resultcard')
       .assert.urlEquals('http://localhost:8000/people/smgc-people-38764')
+      .waitForElementVisible('.record-top__title', 1000)
```

Comment:
@Danwhy you can set a _global_ value for Nightwatch to wait for e.g: https://github.com/dwyl/learn-nightwatch/blob/master/nightwatch.conf.BASIC.js#L23 that way you can leave the second argument out and nightwatch will wait the global amount as a maximum.